scripts/install/install_unix.sh

```diff
@@ -623,6 +625,36 @@ dependencies_check()
   succeeded
 }
 
+# This will check if the required collector user exists when BDOT_SKIP_RUNTIME_USER_CREATION is set to true.
+user_check()
```

Comment:
Check for either user. When the package is installed, it will migrate the legacy user to `bdot` if necessary.